components/common/buttons/ButtonClose.tsx

```diff
@@ -0,0 +1,16 @@
+import type { FC } from 'react';
+
+import CloseIcon from '@mui/icons-material/Close';
+import IconButton, { type IconButtonProps } from '@mui/material/IconButton';
+
+const ButtonCloseComponent: FC<IconButtonProps> = props => {
+  const { 'aria-label': ariaLabel, size, ...restProps } = props;
+
+  return (
+    <IconButton {...restProps} aria-label="delete">
```

Comment:
It looks like `aria-label="delete"` is not the right option. For `IconButton`, it should be something like "Close button". For `CloseIcon`, it should be something like "Close icon".

A better option here will be introducing an id variable and adding it to the label value. The main reason to do so is that `ButtonCloseComponent` can be used across the app in many places. If we use a static value for `aria-label,` all buttons will have the same label. As a result, problems with **a11y**.

Reply:
'+